.env.production

```diff
@@ -1 +1 @@
-VITE_CLOUD_URL='https://console.greptime.cloud/'
\ No newline at end of file
+VITE_CLOUD_URL='https://dev.greptime-cloud-frontend.pages.dev'
```

Comment:
use same url with staging

Reply:
We would need to use 'npm run build' and do some local test, which will automatically use 'production' env. It's better it stays this way for now.